index.xml

```diff
@@ -14990,10 +14990,9 @@ d.type = et;
           <dl>
             <dt id='ref-ECMA-262'>[ECMA-262]</dt>
             <dd>
-              <cite><a href="http://wiki.ecmascript.org/doku.php?id=harmony:specification_drafts">ECMAScript Language Specification, 6th Edition</a></cite>,
-              A. Wirfs-Brock, Editor. Ecma International, 8 November 2013 (Working Draft).
-              <span class="print"><br/>Available at http://wiki.ecmascript.org/doku.php?id=harmony:specification_drafts</span>
-              <br/>Unofficial HTML version available at <a href="http://people.mozilla.org/~jorendorff/es6-draft.html">http://people.mozilla.org/~jorendorff/es6-draft.html</a>.
+              <cite><a href="https://tc39.github.io/ecma262/">ECMAScript 2017 Language Specification</a></cite>,
```

Comment:
The undated spec reference sounds good to me. (We've already had issues with link anchor stability pointing to the ES6 draft, so I hopefully that problem shouldn't get any worse.)